code/modules/projectiles/guns/energy/plasma/plasma.dm

```diff
@@ -127,7 +127,7 @@
 	var/explosion_f_size = 5
 
 	init_firemodes = list(
-		list(mode_name="Super-heavy Plasma", mode_desc="A massive plasma bolt to melt even plasteel", projectile_type=/obj/item/projectile/plasma/heavy/super_heavy, fire_sound='sound/weapons/energy/pulse.ogg', fire_delay=5, icon="kill", projectile_color = "#FFFF00"),
+		list(mode_name="Maraud", mode_desc="A plasma toroid with enough energy behind it to melt plasteel.", projectile_type=/obj/item/projectile/plasma/heavy/super_heavy, fire_sound='sound/weapons/energy/pulse.ogg', fire_delay=5, icon="kill", projectile_color = "#33ff00"),
```

Comment:
What's Maraud mean?

Reply:
Maraud
/məˈrɔːd/
(to) raid and plunder (a place).

In apt reference to [a real-life plasma weapon](https://en.wikipedia.org/wiki/MARAUDER)

Reply:
Huh, I knew of Marauder, but I didn't know it was a word on it's own.